app/src/main/java/me/quadphase/qpdex/databaseAccess/PokemonFactory.java

```diff
@@ -296,27 +403,10 @@ public void getAllDetailedPokemon(){
 
             if(PRINT_DEBUG)
                 Log.d("QPDEX",String.format("Building %s",i));
-//                //We have to slow down the loop intentionally...
-//                try {
-//                    Thread.sleep(10);
-//                }
-//                catch(InterruptedException e){
-//
-//                }
         }
 
     }
 
-    //TODO: FIX thread classes before calling again.
-    public void getAllDetailedPokemonInParallel(){
-        Thread masterBuilder = new DetailedListMaster();
-        masterBuilder.start();
-        try {
-            masterBuilder.join();
-        } catch (InterruptedException e) {
-            e.printStackTrace();
-        }
-    }
 
     private void buildPartOfDetailedPokemonList(int start, int end){
         for (int i = start; i < end; i++) {
```

Comment:
Now that you mention it, It's unlikely we'll build MinimalPokemon for anything except the 721 that are loaded into lists. Alt-forms are distinguished only by their stats. I'll remove both methods to avoid querying the database that way.